sdk/formrecognizer/Azure.AI.FormRecognizer/samples/V3/Sample7_CopyCustomModel.md

```diff
@@ -20,7 +20,7 @@ You can set `endpoint` and `apiKey` based on an environment variable, a configur
 ### Source client
 The source client that contains the custom model we want to copy.
 
-```C# Snippet:FormRecognizerSampleCreateCopySourceClient
+```C# Snippet:FormRecognizerSampleCreateCopySourceClientV3
```

Comment:
I am not sure if V3 would be appropriate, should we use the date version for the folder name?
And would that be enough without having to append the V3 in `FormRecognizerSampleCreateCopySourceClientV3` ?

Reply:
in this case, V3 is reflecting the SDK version. It is following the [structure](https://github.com/Azure/azure-sdk-for-net/tree/feature/formrecognizerv4/sdk/formrecognizer/Azure.AI.FormRecognizer/samples) we talked about of having samples in V3 folder and current samples in either a V4 folder or in the main samples folder.

Although snippet strings are internal to us, they are the way for us to know where the code for the sample lives. We have also modified the samples code and moved all "old" samples to a V3 folder
https://github.com/Azure/azure-sdk-for-net/tree/feature/formrecognizerv4/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples

Do you think this is clear enough for future developers to know which code/snippet/sample, lives where?

Reply:
Hmm... I feel v3 or v4 is awkward since we havent used these terms before. Could we use similar wording as what python did `v3.1` and `v3.2-beta` 

Do you think this would be better?

Reply:
I like it! I will change it to `v3.1` everywhere to make it more clear. I will do it in the next PR to keep this one "small-ish"